src/main/java/com/gtnewhorizons/angelica/transform/RedirectorTransformer.java

```diff
@@ -413,18 +408,27 @@ else if ((node.getOpcode() == Opcodes.GETFIELD || node.getOpcode() == Opcodes.PU
                             }
                             changed = true;
                         }
-
                     }
                 }
             }
-            if (ASSERT_MAIN_THREAD && redirectInMethod && !transformedName.startsWith(SplashProgress) && !(transformedName.startsWith(MinecraftClient) && ExcludedMinecraftMainThreadChecks.contains(mn.name))) {
+            if (ASSERT_MAIN_THREAD && redirectInMethod && !isExcludeFromMainThreadAssert(transformedName, mn.name)) {
                 mn.instructions.insert(new MethodInsnNode(Opcodes.INVOKESTATIC, GLStateManager, "assertMainThread", "()V", false));
             }
         }
 
         return changed;
     }
 
+    private static boolean isExcludeFromMainThreadAssert(String className, String methodName) {
+        if (className.startsWith("cpw.mods.fml.client.SplashProgress")) {
```

Comment:
I'd rather keep these as static final constants

Reply:
Sure! I moved them cause they are the only place where we use classnames with "." Whereas everywhere else we use classnames with "/"